### PR TITLE
(nit) sparse tutorial typos

### PR DESCRIPTION
**Pull request by sygi**

### CODE REVIEW

File: doc/tutorial/sparse.txt [hunk 1]
```diff
@@ -56,10 +56,10 @@ Compressed Sparse Format
 
 .. Changes to this section should also result in changes to library/sparse/index.txt.
 
-Theano supports two *compressed sparse formats*  ``csc`` and ``csr``, respectively based on columns
+Theano supports two *compressed sparse formats*: ``csc`` and ``csr``, respectively based on columns
 and rows. They have both the same attributes: ``data``, ``indices``, ``indptr`` and ``shape``.
 
-  * The ``data`` attribute is a one-dimentionnal ``ndarray`` which contains all the non-zero
+  * The ``data`` attribute is a one-dimesional ``ndarray`` which contains all the non-zero
```
Comment (nouiz) [inline doc/tutorial/sparse.txt]: dimesional -> dimensional

otherwise it looks good. thanks.

Comment (sygi) [inline doc/tutorial/sparse.txt]: corrected, sorry to commit a typo while correcting one :stuck_out_tongue:

**Comment by sygi** [comment]
reopening to rerun travis test

**Comment by nouiz** [comment]
thanks

**Pull request merged**